blockchain/chain_test.go

```diff
@@ -131,9 +131,9 @@ func TestCalcSequenceLock(t *testing.T) {
 	}
 	defer teardownFunc()
 
-	// Since we're not dealing with the real block chain, disable
+	// Since we're not dealing with the real block chain, remove
 	// checkpoints and set the coinbase maturity to 1.
-	chain.DisableCheckpoints(true)
+	chain.RemoveCheckpoints()
```

Comment:
As mentioned elsewhere, should be removed in favor of initial setup with no checkpoints.